automated/linux/kselftest/kselftest.sh

```diff
@@ -151,7 +151,8 @@ if [ -n "${SKIPLIST}" ]; then
     # shellcheck disable=SC2086
     for test_name in ${SKIPLIST}; do
         # shellcheck disable=SC2086
-        sed -i "/.\/${test_name}/c\echo \"selftests: ${test_name} [SKIP]\"" run_kselftest.sh
+        sed -i "/${test_name}/d" run_kselftest.sh
```

Comment:
Looking at run_kselftest.sh (https://git.io/fjgG2), I notice that when there are a list of tests, all but the last ones end the line with "\\". So, if the test being skipped is the last in the list, this will not work.

Reply:
Also, the other case I wonder if you tested, is what if the last test was removed from a test section. Will it work?